weatherApp/Features/WeatherDetails/Data/CoreData Models/DailyTemperatureCD+CoreDataClass.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  DailyTemperatureCD+CoreDataClass.swift
+//  
+//
+//  Created by Dario Nikolic on 19/08/2020.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(DailyTemperatureCD)
+public class DailyTemperatureCD: NSManagedObject {
```

Comment:
Please do not use abbreviations as CD, use whole words (I presume CoreData?). I know that renaming CoreData entities can be painful ( :) ) but please try to do that.